packages/visx-demo/src/pages/geo-albers-usa.tsx

```diff
@@ -8,6 +8,7 @@ const GeoAlbersUsaPage = () => {
   return (
     <Show
       events
+      /* @ts-ignore */
```

Comment:
hmm, might be able to remove this if we make the component type more specific. if this doesn't work no worries.

```tsx
// current
const GeoAlbersUsa = ({ width, height, fullSize = true }: GeoAlbersUsaProps) => { ... }

// proposed
const GeoAlbersUsa: React.FC<GeoAlbersUsaProps> = ({ width, height, fullSize = true }) => { ... }
```